### ISSUE TITLE
Unit tests failing for R:release

### ISSUE THREAD

**Issue by cosi1** [Roche/ggtips#33]
## Observed behavior:

```
> test_check("ggtips")
── 1. Failure: isNullUnit() (@test_grobs.R#70)  ────────────────────────────────
ggtips:::isNullUnit(unit(1, "null")) isn't true.
── 2. Error: isNullUnit() (@test_grobs.R#71)  ──────────────────────────────────
object 'unit.list' not found
Backtrace:
 1. testthat::expect_true(...)
 6. grid:::unit.list
 7. base::get(name, envir = asNamespace(pkg), inherits = FALSE)
```

## Environment:

```
R version 4.0.0 (2020-04-24) -- "Arbor Day"
Copyright (C) 2020 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)
```

**Comment by mjakubczak** [comment]
@cosi1 I can take care of it because my test is failing (due to changes in the latest grid version)

**Comment by cosi1** [comment]
@mjakubczak Thank you :D

**Comment by cosi1** [comment]
Fixed with #35
